opensearch/src/main/java/org/opensearch/sql/opensearch/executor/OpenSearchExecutionEngine.java

```diff
@@ -246,5 +252,16 @@ private void registerOpenSearchFunctions() {
         (builder, args) ->
             builder.makeCall(new GeoIpFunction(client.getNodeClient()).toUDF("GEOIP"), args);
     PPLFuncImpTable.INSTANCE.registerExternalFunction(BuiltinFunctionName.GEOIP, geoIpImpl);
+
+    AggTransferFunctionMap.AGG_FUNCTION_MAP.put(
```

Comment:
I think can move AGG_FUNCTION_MAP to internal of PPLFuncImpTable.
Than we could register agg by
 ```
PPLFuncImpTable.INSTANCE.registerExternalFunction(BuiltinFunctionName.DISTINCT_COUNT_APPROX, distinctCountImpl);
```